src/AutoRest.CSharp/Common/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -48,6 +48,21 @@ public static async Task RunAsync(CommandLineOptions options)
             {
                 var json = await File.ReadAllTextAsync(tspInputFile);
                 var rootNamespace = TypeSpecSerialization.Deserialize(json) ?? throw new InvalidOperationException($"Deserializing {tspInputFile} has failed.");
+
+                // Update usage for Mgmt
+                // TODO: Consolidate InputTypeUsageProvider and SchemaUsageProvider
```

Comment:
look into TCGC to resolve this there

Reply:
This has been addressed in https://github.com/Azure/typespec-azure/issues/916, waiting for a new release of TCGC.

Reply:
I will revert the related to fix the usage in this PR, and updating TCGC will fix the usage.
Don't think this PR merge is pending on the TCGC upgrading since we are not officially supporting TSP input for MPG.